ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -3102,6 +3105,9 @@ tests:
         and not @fips'
       TAG_VERSION: '@4.13'
       TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~ConnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&
+    leases:
```

Comment:
I think need the install workflow's owner to update the job, so I guess we can leave this one untouched?

Reply:
So do you mean we do not change all agent jobs but let them change it, or let them review it?

Reply:
I see in agent disconnected job, also used   - chain: vsphere-provision-bastionhost, so I added explicit lease to all jobs with cluster_profile: vsphere-dis-2

Reply:
okay, if that, i am okay with it.